Docs/Book/RDKit_Book.rst

```diff
@@ -2701,6 +2701,206 @@ path) enumeration algorithm used in the RDKit fingerprint. After a subgraph has
+The name given the macro atom in the main CTAB must match one of the names in one of the templates with the correct class.
+The NATREPLACE attribute specifies the natural replacement for this macro atom, and is ignored by this treatment.  Example:
+
+    # M  V30 TEMPLATE 1 SUGAR/Rib/R NATREPLACE=SUGAR/R
+
+Main Template CTAB and SGROUPs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After the TEMPLATE is a full CTAB with the atoms and bonds of the template.   Each CTAB must contain an SGROUP for the main macro definition atoms and bonds, 
+and one for each leaving group.  The main SGROUP for the template must have a LABEL attribute that is the same as one of the names in the TEMPLATE line, and 
+it must have a CLASS attribute that matches the TEMPLATE line class.
+
+In addition, the main SGROUP must have ATOMS, XBONDS, NATREPLACE attributes.  ATOMS is a list of all atom IDs (1-offset).  XBONDS are the IDs of the connecting 
+bonds.   NATRELACE is the natural replacement as discussed above.  The XBONDS and NATREPLACE attributes are ignored by this treatment.  
+
+The main SGROUP must also contain a list of SAP (SGROUP attachment points) attributes.  Each one is enclosed in parentheses, and starts with a “3”.  
+That is followed by two atom IDs of a bond that starts in this SGROUP and goes to an atom NOT in this SGROUP,  and this is followed by a string indicating 
+the connection name.  The connection name, as discussed above, is usually one of “Al”, “Br”, “Cx” or “Ch”.  SAPs “Al”, “Br”, “Cx” will have one SAP, and 
+“Ch” will typically have two or three distinct SAP attributes.  The order of the SAP attributes for hydrogen bonds (“Ch”) matters, and should go from the H-bond 
+atom nearest to the connecting “Al” atom to the most distant.  Hydrogen bonds do not have a leaving group, so the second ID of the designation is “0”.
+
+Example:
+
+    # M  V30 2 SUP 2 ATOMS=(8 1 2 3 4 5 6 7 8) XBONDS=(1 7)-
+    M  V30 LABEL=U CLASS=BASE SAP=(3 4 9 Al) SAP=(3 8 0 Ch) SAP=(3 6 0 Ch) SAP=(3 7 0 Ch)-
+    NATREPLACE=BASE/U
+
+In addition to the main SGROUP, there will be an SGROUP that identifies each leaving group.  Most leaving groups have one atom, but they could have multiple 
+atoms.  The leaving group SGROUPS have ATOMS, XBONDS, LABEL, and CLASS attributes.   The XBONDS attribute is ignored in this treatment.   LABEL must be 
+“LGRP” (leaving group).
+
+Example:
+
+    # M  V30 1 SUP 1 ATOMS=(1 11) XBONDS=(1 11) LABEL=H CLASS=LGRP
+
+Parsing SCSR files and text blocks
+==================================
+
+The following calls parse the SCSR mol or block produces an SCSRMol:
+
+  * SCSRMolFromScsrDataStream  (stream to SCSRMol)
+  * SCSRMolFromScsrBlock  (text block to SCSRMol)
+  * SCSRMolFromScsrFile  (file to SCSRMol)
+  
+Full Example of SCST files:
+===========================
+
+Here are two examples of SCST files:
+
+`DnaTest3 </Docs/Book/data/DnaTest3.mol>`_.
+
+`CrossLink.mol </Docs/Book/data/CrossLink.mol>`_.
```

Comment:
These links do not work in the final documentation.

You either need to figure out how to make Sphinx copy the files and what the format of the links should be or include links to where the files will be in Github, like this:
(i.e. ``)
```suggestion
`DnaTest3.mol <https://github.com/rdkit/rdkit/blob/master/Docs/Book/data/DnaTest3.mol>`_

`CrossLink.mol <https://github.com/rdkit/rdkit/blob/master/Docs/Book/data/CrossLink.mol>`_
```

Reply:
I made these changes as suggested.  That link does not work for me either because those files are not in master yet